Tests/SuperLinq.Async.Test/CountDownTest.cs

```diff
@@ -10,19 +12,15 @@ public void IsLazy()
 	}
 
 	[Fact]
-	public Task WithNegativeCount()
+	public void ExceptionOnNegativeCount()
```

Comment:
Maybe use zero and make `NonPositiveCount`, or add another test?

Reply:
thanks!